src/networking/responses/checkout-calculate-tax-response.ts

```diff
@@ -22,11 +22,7 @@ export interface CheckoutCalculateTaxResponse {
   tax_amount_in_micros: number;
   total_excluding_tax_in_micros: number;
   tax_inclusive: boolean;
-  pricing_phases: {
-    base: {
-      tax_breakdown: TaxBreakdown[];
-    };
-  };
```

Comment:
@tonidero any concerns about this? This property is being replaced by `tax_breakdown`. The old `pricing_phases` is still returned by the API, but it's been marked as deprecated. Since this is a private interface, I think we can directly remove it.